pallets/frequency-tx-payment/src/lib.rs

```diff
@@ -324,6 +369,29 @@ where
 		Ok((fee.into(), InitialPayment::Capacity))
 	}
 
+	/// Withdraws transaction fee paid Capacity using a key associated to an MSA of a batch of calls.
+	fn withdraw_capacity_fee_batch(
```

Comment:
We separated `withdraw_capacity_fee_batch` into a separate function to get `pay_with_capacity_batch_all` working. Combining functionality so that `withdraw_capacity_fee` can handle both `pay_with_capacity` and `pay_with_capacity_batch_all` extrinsic calls would be a nice to have